tests/test_unitary_rcwe.py

```diff
@@ -0,0 +1,63 @@
+from typing import Dict, List
+
+import numpy as np
+from qiskit.quantum_info import random_unitary
+
+from tests.test_unitary_base import BaseTestUnitary
+
+
+class TestRelativeCleanWastingEntangled(BaseTestUnitary):
+    _controls_no_list: List[int] = [0]
```

Comment:
```suggestion
    _controls_no_list: List[int] = [0, 1]
```
there are two cases in here

Reply:
yes, i forgot to open them.